src/components/ResultsSort.js

```diff
@@ -1,20 +1,32 @@
-import { useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
 import CustomSelect from './CustomSelect';
+import { getMoviesThunk, setSortBy } from '../app/appSlice';
 import '../styles/ResultsSort.scss';
 
-const optionsList = ['release date', 'title', 'rating'];
-
 export default function ResultsSort() {
-  const [selectedOption, setSelectedOption] = useState(optionsList[0]);
+  const sortByList = useSelector(state => state.app.sortByList);
```

Comment:
sortByList, filterBy,  sortBy are verbs. Please use verbs only for methods and functions and nouns for keys and constants. Remove cases where word can be counted as noun or verb by context.